canonical-web-and-design/ubuntu.com
# Build basic data structure and template

## Done

- Change name of function from `cube_home` to `cube_microcerts`
- Add basic ideal data structure 
- Integrate mock data into table in template at `/cube/microcerts`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/cube
- Check view.py mock data is doable with data from API's


## Issue / Card

Fixes [#8818](https://github.com/canonical-web-and-design/ubuntu.com/issues/8818)
